pymeasure/instruments/keysight/keysightB2911B.py

```diff
@@ -0,0 +1,44 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2025 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+import logging
+from .keysightB2901B import KeysightB2901B
+
+log = logging.getLogger(__name__)
+log.addHandler(logging.NullHandler())
+
+
+class KeysightB2911B(KeysightB2901B):
+    """
+    This represents the Keysight B2911B Source-Measure Unit interface.
+
+    .. code-block:: python
+
+        smu = KeysightB2911B(address)
+
+    """
+
+    def __init__(self, adapter, name="Keysight B2911B Source-Measure Unit", **kwargs):
+        super().__init__(
+            adapter, name, **kwargs)
```

Comment:
This instrument does not provide any advantage, consider creating a "series" driver which works for several devices (adjusting the docstring accordingly).